card/mod/02_basic_types/spec/set/all/base_spec.rb

```diff
@@ -1,26 +1,41 @@
 # -*- encoding : utf-8 -*-
 
 describe Card::Set::All::Base do
-
   describe 'handles view' do
+    describe 'name view' do
+      it('name') { expect(render_card(:name)).to eq('Tempo Rary') }
+      it 'pluralizes' do
+        name = render_content '{{Joe User|name; variant: pluralize}}'
+        expect(name).to eq('Joe Users')
+      end
+      it 'singularizes' do
+        name = render_content '{{Sunglasses|name; variant: singularize}}'
+        expect(name).to eq('Sunglass')
+      end
+      it 'handles more than one variant' do
+        name = render_content(
+          '{{Sunglasses|name; variant: singularize, upcase}}'
+        )
+        expect(name).to eq('SUNGLASS')
+      end
+    end
 
-    it("name"    ) { expect(render_card(:name)).to      eq('Tempo Rary') }
-    it("key"     ) { expect(render_card(:key)).to       eq('tempo_rary') }
-    it("linkname") { expect(render_card(:linkname)).to  eq('Tempo_Rary') }
+    it('key'     ) { expect(render_card(:key)).to       eq('tempo_rary') }
```

Comment:
Put one space between the method name and the first argument.<br>Space inside parentheses detected.